src/libtest/lib.rs

```diff
@@ -435,15 +437,41 @@ Test Attributes:
              usage = options.usage(&message));
 }
 
+// FIXME: Copied from libsyntax until linkage errors are resolved.
```

Comment:
Is this FIXME something you intend to fix in this PR? If not, can you file an issue and include the issue number here.